OpenRA.Mods.Common/Traits/Conditions/ConditionManager.cs

```diff
@@ -219,23 +187,12 @@ public int RevokeCondition(Actor self, int token)
 		}
 
 		/// <summary>Returns true if the given external condition will have an effect on this actor.</summary>
-		public bool AcceptsExternalCondition(Actor self, string condition, bool timed = false)
+		public bool AcceptsExternalCondition(Actor self, string condition)
 		{
 			if (state == null)
 				throw new InvalidOperationException("AcceptsExternalCondition cannot be queried before the actor has been fully created.");
 
-			if (!externalConditions.Contains(condition))
-				return false;
-
-			// A timed condition can always replace an existing timed condition (resetting its duration)
-			if (timed && timers.ContainsKey(condition))
```

Comment:
@GraionDilach: If you want to cap effect stacking, make `Targetable` require a condition to be less than / at most a certain amount. That would also give the actors & players feedback by preventing useless attempts.